winforms/src/toga_winforms/window.py

```diff
@@ -181,17 +190,73 @@ def hide(self):
     # Window state
     ######################################################################
 
-    def set_full_screen(self, is_full_screen):
-        if is_full_screen:
-            self.native.FormBorderStyle = getattr(WinForms.FormBorderStyle, "None")
-            self.native.WindowState = WinForms.FormWindowState.Maximized
-        else:
+    def get_window_state(self):
+        window_state = self.native.WindowState
+        if window_state == WinForms.FormWindowState.Maximized:
+            if self.native.FormBorderStyle == getattr(WinForms.FormBorderStyle, "None"):
+                if self._in_presentation_mode:
+                    return WindowState.PRESENTATION
+                else:
+                    return WindowState.FULLSCREEN
+            else:
+                return WindowState.MAXIMIZED
+        elif window_state == WinForms.FormWindowState.Minimized:
+            return WindowState.MINIMIZED
+        elif window_state == WinForms.FormWindowState.Normal:
+            return WindowState.NORMAL
+        else:  # pragma: no cover
+            # Marking this as no cover, since the above cases cover all the possible values
+            # of the FormWindowState enum type that can be returned by WinForms.WindowState.
+            return
```

Comment:
As previously - you avoid the pragma by using one of the actual cases as the bucket case.

Reply:
Changed it.